src/Raven.Server/Properties/Settings/settings.posix.json

```diff
@@ -1,5 +1,5 @@
 ﻿{
     "ServerUrl": "http://127.0.0.1:0",
     "Setup.Mode": "Initial",
-    "DataDir": "$HOME/raven"
+    "DataDir": "RavenData"
```

Comment:
Where does this go?

Reply:
ravendb@ravendb:~/RavenDB/Server $ ls -ltr RavenData/
total 4
drwxr-xr-x 3 ravendb ravendb 4096 Dec 18 12:50 System
ravendb@ravendb:~/RavenDB/Server $

Reply:
relative..

Reply:
relative to what?
So if I open this from a different path I get different data?

Reply:
no.  it will be always in the assembly's directory: i.e.:
```
ravendb@ravendb:~/RavenDB $ ./Server/Raven.Server --ServerUrl=http://0.0.0.0:8080 --Security.UnsecuredAccessAllowed=PublicNetwork
       _____                       _____  ____ 
      |  __ \                     |  __ \|  _ \ 
      | |__) |__ ___   _____ _ __ | |  | | |_) |
      |  _  // _` \ \ / / _ \ '_ \| |  | |  _ < 
      | | \ \ (_| |\ V /  __/ | | | |__| | |_) |
      |_|  \_\__,_| \_/ \___|_| |_|_____/|____/ 


      Safe by default, optimized for efficiency

 Build 40, Version 4.0, SemVer 4.0.0-custom-40, Commit d7047cd
 PID 2952, 32 bits, 4 Cores, Phys Mem 927.32 MBytes, Arch: Arm
 Source Code (git repo): https://github.com/ravendb/ravendb
 Built with love by Hibernating Rhinos and awesome contributors!
+---------------------------------------------------------------+
Using GC in server non concurrent mode retaining memory from the OS.
Server available on: http://ravendb:8080
Tcp listening on 0.0.0.0:8471
Server started, listening to requests...
TIP: type 'help' to list the available commands.
Running non-interactive.
^C

ravendb@ravendb:~/RavenDB $ ls -ltra Server/RavenData/
total 32
drwxr-xr-x 4 ravendb ravendb 20480 Dec 18 14:19 ..
drwxr-xr-x 3 ravendb ravendb  4096 Dec 18 14:19 .
drwxr-xr-x 3 ravendb ravendb  4096 Dec 18 14:19 System
ravendb@ravendb:~/RavenDB $ 
```

running for the first time from `RavenDB` : `./Server/Raven.Server`
the "working directory" is still `RavenDB/Server`